src/main/java/com/cs56fitnessapp/models/workout/StrengthTraining.java

```diff
@@ -0,0 +1,45 @@
+package com.cs56fitnessapp.models.workout;
+
+import com.cs56fitnessapp.models.User;
+import java.time.LocalDate;
+
+/**
+ *  @author Daniel Cervantes
+ *  @author Ksenia Koldaeva
+ *
+ * Date created: 10/18/17
+ * Last updated: 10/31/17
+ */
+
+public abstract class StrengthTraining extends Workout {
+
+    /**
+     * Constructs Strength Training object with provided parameters
+     * @param user user performing Strength Training
+     * @param date date of Strength Training
+     * @param warmUpTimeHrs time of warm up
+     * @param coolDownTimeHrs time of cool down
+     * @param timePerformingHours amount of time of Strength Training in hours
+     * @param met_Value workout intensity
+     */
+
+    public StrengthTraining(User user, LocalDate date, int warmUpTimeHrs, int coolDownTimeHrs, double timePerformingHours, double met_Value) {
+        super(user, date);
+        this.setWarmUpTimeHrs(warmUpTimeHrs);
+        this.setCoolDownTimeHrs(coolDownTimeHrs);
+        this.setTimePerformingHours(timePerformingHours);
+        this.setMET_value(met_Value);
+    }
+
+    /**
+     * Will be using caloriesOutByMET() to calculate calories out
+     * based on the MET level the user gives
+     * @return calories out from Strength Training
+     */
+
+    public int getCaloriesOutByMet() {
+        return FitnessFormulas.caloriesOutByMET(this.getMET_value(), this.getUser().getbodyMassKg(), this.getTimePerformingHours());
+    }
+
+
```

Comment:
`getCaloriesOutByMet()` has correct logic and will work. 
However, I do not think you would as ask the user to provided `metValue` directly. That is why we should not probably have this property in `Workout` class. 
I was implementing the same logic in a way, that user has to provide swimming stroke for swimming from a predetermined selection. (e.g. breaststroke, freestyle, etc) and cycling style (mountain,  leisure, racing) for cycling.
This is realized via having an `enum` for options. Then, this option is passed to a constructor of a concrete class. Following that, we'll do dropdowns in the view.

Reply:
I suggest for you to do an enum for `StrenghtTrainingEffort` (or other descriptive name) with two options - HEAVY and LIGHT and then use these options to split the logic in  your `getCaloriesOut` method.

Reply:
Note: your public method for getting caloric value for strength workout must be declared as following to implement the interface: 
```java
 @Override
    public int getCaloriesOut() {}
```